code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -338,6 +338,10 @@ Credit where due:
 	CLOCKCULTCHANGELOG\
 	</ul>\
 	<hr>\
+	<li><b>Brass Flasks:</b> These contain 30u of Zelus oil, this oil can be used to help stun people, and more son on Blood Cult\
```

Comment:
```suggestion
	<li><b>Zelus oil:</b> A new reagent. It can be used to heal the faithful to Ratvar, or kill heretics and moreso stun blood cultists,\
```